program/idl.json

```diff
@@ -5345,6 +5345,84 @@
           }
         ]
       },
+      {
+        "kind": "instructionNode",
+        "name": "createNativeMint",
+        "docs": [
+          "Creates the native mint.",
+          "",
+          "This instruction only needs to be invoked once after deployment and is",
+          "permissionless. Wrapped SOL (`native_mint::id()`) will not be",
+          "available until this instruction is successfully executed."
+        ],
+        "optionalAccountStrategy": "programId",
+        "accounts": [
+          {
+            "kind": "instructionAccountNode",
+            "name": "payer",
+            "isWritable": true,
+            "isSigner": true,
+            "isOptional": false,
+            "docs": ["Funding account (must be a system account)"]
+          },
+          {
+            "kind": "instructionAccountNode",
+            "name": "nativeMint",
+            "isWritable": true,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["The native mint address"]
+          },
+          {
+            "kind": "instructionAccountNode",
+            "name": "systemProgram",
+            "isWritable": false,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["System program for mint account funding"],
+            "defaultValue": {
+              "kind": "publicKeyValueNode",
+              "publicKey": "11111111111111111111111111111111"
+            }
+          }
+        ],
+        "arguments": [
+          {
+            "kind": "instructionArgumentNode",
+            "name": "discriminator",
+            "defaultValueStrategy": "omitted",
+            "docs": [],
+            "type": {
+              "kind": "numberTypeNode",
+              "format": "u8",
+              "endian": "le"
+            },
+            "defaultValue": {
+              "kind": "numberValueNode",
+              "number": 31
+            }
+          }
+        ],
+        "remainingAccounts": [
+          {
+            "kind": "instructionRemainingAccountsNode",
+            "isOptional": true,
+            "isSigner": true,
+            "docs": [],
+            "value": {
+              "kind": "argumentValueNode",
+              "name": "multiSigners"
+            }
+          }
+        ],
```

Comment:
[According to the Rust code](https://github.com/solana-labs/solana-program-library/blob/2d795f15287f6a0907bfe68f3fa252e30ee9572b/token/program-2022/src/instruction.rs#L609-L620), this instruction does not use additional remaining signers to multisigs. So we can safely remove this.

Reply:
Thanks! Are you able to rebase the PR when you have a moment?

Reply:
just did that right now, removed the multi signer support LGTM!

Reply:
@lorisleiva